webapi/getfeeaddress.go

```diff
@@ -19,6 +19,8 @@ import (
 // addrMtx protects getNewFeeAddress.
 var addrMtx sync.Mutex
 
+const defaultMinRelayTxFee = dcrutil.Amount(1e4)
```

Comment:
Since the constant is only being used in `getCurrentFee` the scope could be reduced to just that function?